sdk/monitor/Azure.Monitor.Query/tests/DateTimeRangeTests.cs

```diff
@@ -39,5 +39,14 @@ public void ToStringStartEndTests()
 
             Assert.AreEqual("2021-01-02T03:04:05.0000000Z/2021-05-04T03:02:01.0000000Z", new DateTimeRange(startTime, endTime).ToString());
         }
+
+        [TestCase("PT23M")]
+        [TestCase("PT23M/2021-05-04T03:02:01.0000000Z")]
+        [TestCase("2021-05-04T03:02:01.0000000Z/PT23M")]
+        [TestCase("2021-01-02T03:04:05.0000000Z/2021-05-04T03:02:01.0000000Z")]
+        public void CanRoundtrip(string range)
```

Comment:
Do we have a test for the exception cases?

Reply:
We do now.